velox/common/base/BitUtil.h

```diff
@@ -958,6 +958,21 @@ inline void padToAlignment(
   }
 }
 
+#if defined __has_builtin
+#if __has_builtin(__builtin_bswap128)
+#define HAS_BUILTIN_BSWAP_INT128 1
+inline __int128_t builtin_bswap128(__int128_t value) {
```

Comment:
The signature of the function is the same in both cases. Would it make sense to move #if-define inside the function?